### ISSUE TITLE
Payload may not be larger than 1048576 bytes

### ISSUE THREAD

**Issue by fabricious13** [AllTheMods/ATM-6#2111]
**All the Mods 6 - ATM6 - v1.7.2 with forge v36.2.2**

I was playing this modpack for 3 months, I updated the modpack to the latest version and I managed to play that day, the next day I couldn't open the refined storage crafting grid giving it automatic crash when trying to open it. I found out where the error was on a refined disk by removing it from the disk drive and thus managing to play, later it was impossible to enter the server with the following error always appearing (attached) – “payload may not be larger than 1048576 bytes” . I've removed this mod and many others, I've tried to use a version of the previous map, reduce the version of mods in general and even the forge, I still removed possible chunks and blocks that could be causing this and nothing concluded. This happens with multiple players trying to join.

1. Refined Storage system with thousands items
2. Multiples players with different refined systems
3. Update to the last versions all the mods
4. Try to enter de server
5. Error appears - payload may not be larger than 1048576 bytes

**Attachment**
https://snipboard.io/Gp8Xus.jpg

![payload](https://user-images.githubusercontent.com/37511127/128118465-7fd21c1a-753b-4370-b819-b90e9d60def3.png)

**Logs**
Does not create any logs or errors in the server console

**Comment by SkillzUNV** [comment]
When we encountered this same error it turned out to be corrupt NBT data on one Resourceful Bee in a Jar stored in an RS system on our server, we discovered it by using the NBT explorer and found an item with a huge amount of NBT data.  Hope this helps.

**Comment by fabricious13** [comment]
Thanks, i tried to remove that mods but doesnt work yet. Can u explain me how did u find that with NBT explorer pls.

**Comment by jebjordan** [comment]
@fabricious13 I recommend you just add the connectivity mod to your client. 
should fix it right up

**Comment by Sakata-MC** [comment]
I've been wondering why there is a payload limit to begin with. Seems with as popular modpacks are that they'd have removed it. Though I'd imagine it exists for *some reason* I'm not sure what it'd be.

This is the mod that was referenced above:
https://www.curseforge.com/minecraft/mc-mods/connectivity
If there are no issues with having such a mod, it might actually be a good one to have in the pack by default. I know in the past I've encountered such issues and before I learned a bit more about NBTedit than I ever wanted to, it would have saved several worlds from being scrapped.
